website/docs/cloud-docs/workspaces/settings/deletion.mdx

```diff
@@ -0,0 +1,49 @@
+---
+page_title: Destruction and Deletion - Workspaces - Terraform Cloud
+description: |-
+  Learn about destroying infrastructure and deleting workspaces in Terraform Cloud.
+---
+
+# Destruction and Deletion
+
+Terraform Cloud workspaces have two primary delete actions:
+
+- [Destroy Infrastructure](#destroy-infrastructure) deletes resources managed by the Terraform Cloud workspace by triggering a destroy run.
+- [Delete Workspace](#delete-workspaces) deletes the workspace itself without triggering a destroy run.
+
+In general, you should perform both actions in the above order when destroying a workspace to ensure resource cleanup for all infrastructure managed by a workspace.
+
+## Destroy Infrastructure
+
+Destroy plans delete the infrastructure managed by a workspace. It's recommended that you destroy the infrastructure managed by a workspace prior to deleting the workspace itself, otherwise the infrastructure resources will continue to exist but will become unmanaged.
+You will need to go into your infrastructure providers to delete the resources manually.
+
+Destroy plans
+
+Before queuing a destroy plan, you must enable the `Allow destroy plans` setting.
+
+### Manually Destroy
```

Comment:
I feel we don't need this section because this is covered above? Unless we can provide more context on when you'd want to automatically v.s. manually destroy?

Reply:
I think we're just trying to define terms here, but I may need to be sold on why they get headings.

Reply:
The structure here currently matches the structure on the workspace destroy settings page itself

Reply:
![image](https://github.com/hashicorp/terraform-docs-common/assets/33099255/e609338e-12ce-4f27-b1cb-9ff09f391046)

But I'm totally game to remove if it's not reducing clarity as opposed to adding

Reply:
I think's more distracting then helpful, also the UI might change in the future, which would make this doubly distracting. I think it's better to group by what the options actually do rather than where they appear on the page.

Reply:
Hmm I'm kind of confused, they are already grouped by functionality both here and on the settings page.

![image](https://github.com/hashicorp/terraform-docs-common/assets/33099255/248ec189-5833-4872-abdc-1515b3665d51)